### DISCUSSION TITLE
Is it possible to start a workflow from a separate host process not running any workers?

### DISCUSSION THREAD

**Discussion by zaytsevand** [danielgerlag/workflow-core#1106] [Q&A] [answered]
The idea is to split a microservice that will start new workflows on call and a separate instance(s) that will be responsible for work.

**Comment by IvanJosipovic** [comment]
Yes it is. I'm doing that in a project with no issues.

**Comment by zaytsevand** [comment, marked as answer]
After seeing another discussion on starting a workflow without registering all the same workflows with the client, I realized that it was exactly my mistake: I was trying to use a client that wasn't set up 'properly'.
For everyone else interested, the conceptual way in v.3.7 to start a workflow from another process will be:
```csharp
workflowController = new WorkflowController(<... same configuration as a host ...>); // important: controller, not host
workflowController.RegisterWorkflow<YourWorkflow>();
await workflowController.StartWorkflow(nameofYourWorkflow, versionofYourWorkflow, data, reference);
```
I wonder if other features like queues or lock mechanisms would improve on that.
